gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagProcessingEngineTest.java

```diff
@@ -54,7 +54,7 @@ public class DagProcessingEngineTest {
   private DagManagementTaskStreamImpl dagManagementTaskStream;
   private DagTaskStream dagTaskStream;
   private DagProcFactory dagProcFactory;
-  private static MostlyMySqlDagManagementStateStore dagManagementStateStore;
+  private static MostlyMySqlDagManagementStateStore testDagManagementStateStore;
```

Comment:
this is ok too, but in all other tests it is named `dagManagementStateStore`

Reply:
compilation was failing when it had the same name as the `DagTask.dagManagementStateStore` field because I made it protected instead of private. I had to distinguish this static var from its internal field being referenced.

Reply:
Does this field need to be static? If so please add a comment as to why

Reply:
I did not follow it. why do you have to distinguish  DagTask.dagManagementStateStore from dagManagementStateStore ??

Reply:
This method wasn't compiling before but if I reference the static field explicitly it works. It's static bc this same reference is used to instantiate every `MockedDagTask`
```
    public MockedDagTask(DagActionStore.DagAction dagAction, boolean isBad) {
      super(dagAction, leaseObtainedStatus, DagProcessingEngineTest.dagManagementStateStore);
      this.isBad = isBad;
    }
```